src/redux/features/participantProjectSpecies/participantProjectSpeciesSelectors.ts

```diff
@@ -5,8 +5,8 @@ import { participantProjectSpeciesCompositeKeyFn } from './participantProjectSpe
 export const selectParticipantProjectSpeciesRequest = (requestId: string) => (state: RootState) =>
   state.participantProjectSpecies[requestId];
 
-export const selectParticipantProjectSpeciesListRequest = (requestId: string) => (state: RootState) =>
-  state.participantProjectSpeciesList[requestId];
+export const selectParticipantProjectSpeciesListRequest = (projectId: number) => (state: RootState) =>
+  state.participantProjectSpeciesList[projectId];
```

Comment:
This is now returning the same as selectParticipantProjectSpeciesRequest. Should this be  state.participantProjectSpeciesList?

Reply:
It is already returning `state.participantProjectSpeciesList[projectId];`. `selectParticipantProjectSpeciesRequest` selects from `state.participantProjectSpecies[requestId];` (line 6). Not sure I follow. Perhaps the slices are crossed up?

Reply:
I think the list need a requestId, but not a projectId

Reply:
Why?

Reply:
After [discussion](https://github.com/terraware/terraware-web/pull/2555#discussion_r1583518035), no need to change this! Sorry for the wrong review!